### DISCUSSION TITLE
[Feature] Installation Instructions That Do Not Mandate Docker?

### DISCUSSION THREAD

**Discussion by n0w33d** [immich-app/immich#3608] [Feature Request]
### The feature

I would like to host my instance of Immich without the need for a Docker VM, I wish to run it bare-metal. I scoured the internet for any information regarding this and the closest I have seen is a buggy [Arch AUR script](https://aur.archlinux.org/packages/immich) that does not correctly install the software or dependencies. Why does this project mandate the use of a Docker container?

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by jrasm91** [comment]
This is an open source project and we don't mandate anything. True, we only provide (free) support for and distribute the docker install method. Docker is an industry wide standard that we have chosen to adopt. At this stage of the product our goal and focus is to improve the stability of the project, expand the feature set, fix bugs, etc., rather than spend time packaging and testing different install methods and mediums.
